pyproject.toml

```diff
@@ -10,9 +10,9 @@ authors = [{name = "metatrain developers"}]
 
 dependencies = [
     "ase < 3.23.0",
-    "metatensor-learn==0.2.3",
-    "metatensor-operations==0.2.3",
-    "metatensor-torch==0.5.5",
+    "metatensor-learn==0.3.0",
+    "metatensor-operations==0.3.0",
+    "metatensor-torch==0.6.0",
```

Comment:
Do we also have to fix the minor version or is it enough to do `>=0.6,<0.7`?

Reply:
Since `metatensor-torch` moves quite fast these days, I would also fix the minor version. For 0.5, almost every minor version broke `metatrain`